surveys/2024-annual-survey/translations/fr.md

```diff
@@ -160,31 +160,31 @@ Type: select all that apply
 
 - Je n'utilise pas de version nightly
```

Comment:
The title for this section should be `### Si vous utilisez une version nightly, veuillez préciser le(s) motif(s).`

Reply:
will fix in a subsequent patch